elasticsearch/transport.py

```diff
@@ -292,7 +294,7 @@ def perform_request(self, method, url, params=None, body=None):
 
         if body is not None:
             try:
-                body = body.encode('utf-8', 'surrogatepass')            
+                body = body.encode('utf-8', 'surrogatepass')
```

Comment:
please configure your editor not to make changes like these. It makes the PR hard to review when they are unrelated stylistic changes in unrelated places.